webapps-schema/src/main/java/io/camunda/webapps/schema/entities/ExporterEntity.java

```diff
@@ -14,6 +14,8 @@
  */
 public interface ExporterEntity<T extends ExporterEntity<T>> {
 
+  String DEFAULT_TENANT_IDENTIFIER = "<default>";
```

Comment:
:wrench: I don't think this should be part of this interface. May be move it to `AbstractExporterEntity`?